code/modules/mob/living/status_procs.dm

```diff
@@ -379,6 +379,16 @@
 		S = apply_status_effect(STATUS_EFFECT_SLEEPING, -1)
 	return S
 
+///////////////////////// CLEAR STATUS /////////////////////////
+
+/mob/living/proc/status_clear(amount)
+	AdjustStun(-60 * amount)
+	AdjustKnockdown(-60 * amount)
+	AdjustUnconscious(-60 * amount)
+	AdjustSleeping(-100 * amount)
+	AdjustParalyzed(-60 * amount)
+	AdjustImmobilized(-60 * amount)
```

Comment:
```suggestion
/mob/living/proc/adjust_status_effects_on_shake_up()
	AdjustStun(-60)
	AdjustKnockdown(-60)
	AdjustUnconscious(-60)
	AdjustSleeping(-100)
	AdjustParalyzed(-60)
	AdjustImmobilized(-60)
```
* Use informative proc names. If it's long, it's long. The important part is for the reader to understand what is going on with the less code checking possible.
* Don't add extra unused arguments and modifiers. If anyone needs them in the future they can easily add them.

Reply:
Got it. Thanks for the tips!